source/about/cloud-subscriptions.rst

```diff
@@ -85,14 +85,14 @@ How is Mattermost Cloud secured?
 
 Mattermost Cloud has encryption at-rest and in-transit. We're SOC2 Type 1 certified and working towards SOC2 Type 2.
 
-Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC (Virtual Private Cloud) dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace. Please contact Mattermost Sales for more information on this option. 
+Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC (Virtual Private Cloud) dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace. Please contact Mattermost Sales for more information on this option.
```

Comment:
is it "on-premises" or "on-premise"?

Reply:
On-premises is the correct way of saying it, but we say "on-premise" in other parts of the docs, and we also say "on-prem" (which is a nice compromise). I'd propose going with on-prem as that's on the website and we can reverse engineer any other instances. Thoughts?